reflex.conf

```diff
@@ -1,2 +1,2 @@
 # backend
--r '^(cmd|pkg)/.*\.go$' -R '^node_modules/' -s -- sh -c 'make bin/ocis-graph-debug && bin/ocis-graph-debug --log-level debug server --debug-pprof --debug-zpages'
+-r '^(cmd|pkg)/.*\.go$' -R '^node_modules/' -s -- sh -c 'make bin/ocis-graph-debug && bin/ocis-graph-debug --log-level debug server --debug-pprof --debug-zpages --oidc-endpoint="https://deepdiver" --oidc-insecure=1'
```

Comment:
https://deepdiver hardcoded? You know that you can export env variables before executing `make watch`?

Reply:
:see_no_evil: 

we need to address the https handling in ocis (some proxy module or caddy or what ever) - then we can get rid of this.

Reply:
You can still already avoid this hardcoded values by using env variables ;)

Reply:
Sure. Will fix